modules/pricer/src/main/java/com/opengamma/strata/pricer/fx/BlackFxVanillaOptionProductPricer.java

```diff
@@ -94,7 +97,7 @@ public CurrencyAmount presentValue(
       BlackVolatilityFxProvider volatilityProvider) {
```

Comment:
Javadoc: The present value of the product is the value on the valuation date. It is expressed in the counter currency.